cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -265,7 +395,7 @@ def single_qubit_randomized_benchmarking(
         num_circuits=num_circuits,
         repetitions=repetitions,
     )
-    return result[qubit]
+    return result._results_dictionary[qubit]
```

Comment:
lets not access private properties. either make it public or implement `ParallelRandomizedBenchmarkingResult. __getitem__ ` then return `result[qubit]`